content/concept-checks.md

```diff
@@ -0,0 +1,103 @@
+
+
+## Usage
+
+Checks typically have the lifecycle described below.
+
+<!--{# Diagram sources: https://docs.google.com/drawings/d/1Ez8OZVB2TLH-b_kSFOAgfYqXlEQt4KaUBW6F3TJAv_Q/edit #}-->
+
+
+[![Check flow diagram (successful cashing)](img/checks-happy_path.png)](img/checks-happy_path.png)
+
+**Step 1:** To create a Check, the sender submits a [CheckCreate][] transaction and specifies the receiver (`Destination`), expiration time (`Expiration`), and maximum amount that may be debited from the sender's account (`SendMax`).
+
+
+**Step 2:** After the CheckCreate transaction is processed, a [Check object](reference-ledger-format.html#check) is created on the XRP Ledger. This object contains the properties of the Check as defined by the transaction that created it. The object can only be modified by the sender (by canceling it with a [CheckCancel][] transaction) or receiver (by canceling it or cashing it) before the expiration time passes. After the expiration time, anyone may cancel the Check.
+
+**Step 3:** To cash the check, the receiver submits a [CheckCash][] transaction. The receiver has two options for cashing the check:
+
+* `Amount` — The receiver can use this option to specify an exact amount to cash. This may be useful for cases where the sender has padded the check to cover possible [transfer fees](concept-transfer-fees.html) and the receiver can only accept the exact amount on an invoice or other contract.
+
+* `DeliverMin` — The receiver can use this option to specify the minimum amount they are willing to receive from the Check. If the receiver uses this option, `rippled` attempts to deliver as much as possible and will deliver at least this amount. The transaction fails if the amount that can be credited to the receiver is not at least this amount.
+
+If the sender has enough funds to cover the Check and the expiration time has not passed, the funds are debited from the sender's account and credited to the receiver's account, and the Check object is is destroyed.
+
+
+
+#### Expiration Case
+
+In the case of expirations, Checks have the lifecycle described below.
+
+<!--{# Diagram sources: https://docs.google.com/drawings/d/1JOgI3H5tpV1yasYe5WLrdxVXLhcQu0bhPfN0mzzS1YQ/edit #}-->
+
+
+[![Check flow diagram (expiration)](img/checks_expiration.png)](img/checks_expiration.png)
+
+
+All Checks start the same way, so **Steps 1 and 2** are the same.
+
+**Step 3a:** If the Check expires before the receiver can cash it, the Check can no longer be cashed but remains in the ledger.
+
+**Step 4a:** After a Check expires, anyone may cancel it by submitting a [CheckCancel][] transaction. That transaction removes the Check from the ledger.  
+
+
+
+## Availability of Checks
+
+Checks require `rippled` v0.90.0 or later and have been enabled on the production XRP Ledger since 2018-03-15.
```

Comment:
Just found out today that Checks haven't been enabled yet on the production ledger. That explains why we couldn't find the EnableAmendment pseudo-tx!

The plan is to start voting in favor of Checks soon (maybe today?), but they won't be enabled for at least 2 more weeks.